core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksUtilities.scala

```diff
@@ -48,12 +48,15 @@ object DatabricksUtilities {
   val Folder = s"/SynapseMLBuild/build_${BuildInfo.version}"
   val ScalaVersion: String = BuildInfo.scalaVersion.split(".".toCharArray).dropRight(1).mkString(".")
 
-  // SynapseML info
-  val Version = s"com.microsoft.azure:synapseml_$ScalaVersion:${BuildInfo.version}"
-  val Repository = "https://mmlspark.azureedge.net/maven"
+  // Package info
+  val Version = PackageUtils.PackageMavenCoordinate
+  val Repository = PackageUtils.PackageRepository
+  val OnnxVersion = PackageUtils.OnnxProtobufMavenCoordinate
+  val OnnxRepository = PackageUtils.OnnxProtobufRepository
 
   val Libraries: String = List(
     Map("maven" -> Map("coordinates" -> Version, "repo" -> Repository)),
+    Map("maven" -> Map("coordinates" -> OnnxVersion, "repo" -> OnnxRepository)),
```

Comment:
need on GPU?

Reply:
I think we can remove ONNXVersion and ONNXRepo from here right?

Reply:
ah, ok, and just let Databricks transitively pull it?  got it, will test